params/params_testnet.go

```diff
@@ -126,4 +127,19 @@ var TestNetParams = Params{
 	HDCoinType: 223,
 
 	//OrganizationPkScript:  hexMustDecode("76a914868b9b6bc7e4a9c804ad3d3d7a2a6be27476941e88ac"),
+
+	// Coins config
+	// TODO The future needs to be redefined or deleted
+	CoinsCfg: &types.CoinConfigs{
```

Comment:
1. For the coinid does not set the `CoinsCfg` explicitly such as the MEER and QIT.  better to create new feeType for them , and add restriction that require all coin need to have a `CoinsCfg` explicitly.
2. For the coinid has `CoinsCfg` such as MET and TER, what happen if set value to zero ? (For example,  set `TER` value is 0, and equalfee),

Reply:
4b98218